### PR TITLE
uboot-mvebu: update to version 2023.07

### PR DESCRIPTION
**Pull request by damnms**
Since 2021.07 multiple bugs where introduced that made it impossible to create a bootable target for mvebu [1].
This is now fixed in 2023.07-rc1.

[1] https://github.com/openwrt/openwrt/issues/11661

Signed-off-by: Oli Ze <olze@trustserv.de>

### CODE REVIEW

**Review by neheb — APPROVED** [review]

**Comment by neheb** [comment]
Off-topic:

ping @hesiath

Should the Helios 4 uboot package on Armbian be updated?

File: package/boot/uboot-mvebu/Makefile [hunk 1]
```diff
@@ -8,10 +8,10 @@
 include $(TOPDIR)/rules.mk
 include $(INCLUDE_DIR)/kernel.mk
 
-PKG_VERSION:=2023.01
+PKG_VERSION:=2023.07-rc1
 PKG_RELEASE:=$(AUTORELEASE)
```
Comment (kkudielka) [inline package/boot/uboot-mvebu/Makefile:12]: This should be `PKG_RELEASE:=1` to avoid merge conflict.

**Comment by Heisath** [comment]
> Off-topic:
> 
> ping @Heisath
> 
> Should the Helios 4 uboot package on Armbian be updated?

Yeah that would be great. We are thinking about retiring the mvebu boards, so one update to a really current version sounds good.

**Comment by damnms** [comment]
i am not available for the next 6-7 weeks. if someone is interested in this, please do it yourself.

**Comment by damnms** [comment]
@kkudielka why not create a PR to openwrt master so everyone can benefit from those changes?

**Comment by damnms** [comment]
i give up, this is too nucking me futs. i might try it again in the next weeks. the good thing is i will switch away from mvebu in the next weeks, so tbh i dont care anymore if this is still broken. it is for over 2 years. seems like no one cares. this is probably a 5 minute thing for a person that is familiar with all that guidelines, rebasing, stashing, bisecting, git whatever stuff, but this is not my environment, not my daily business, and i dont want to waste my time with this.

**Comment by rmilecki** [comment]
Ref: https://github.com/openwrt/openwrt/pull/13167

**Pull request closed without merging**